client/components/TestRenderer/UnexpectedBehaviorsFieldset/index.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { Feedback, Fieldset } from '..';
+
+const ProblemOptionContainer = styled.div`
+    &.enabled {
+        margin-bottom: 15px;
+    }
+`;
+
+const Label = styled.label`
+    display: inline-block;
+    width: 100%;
+    margin-bottom: 5px;
+
+    > select,
+    > input[type='text'] {
+        min-width: 120px;
+        width: 50%;
+        height: 26px;
+        margin-left: 5px;
+    }
+
+    > input[type='checkbox'] {
+        margin-right: 5px;
+    }
+
+    &.off-screen {
+        position: absolute !important;
+        height: 1px;
+        width: 1px;
+        overflow: hidden;
+        clip: rect(1px, 1px, 1px, 1px);
+        white-space: nowrap;
+    }
+`;
+
+const UnexpectedBehaviorsFieldset = ({
+    commandIndex,
+    unexpectedBehaviors,
+    isSubmitted
+}) => {
+    const severityOptions = ['Moderate', 'High'];
+
+    return (
+        <Fieldset id={`cmd-${commandIndex}-problems`}>
+            <legend>{unexpectedBehaviors.description[0]}</legend>
+            {isSubmitted && (
+                <Feedback
+                    className={`${
+                        unexpectedBehaviors.description[1].required &&
+                        'required'
+                    } ${
+                        unexpectedBehaviors.description[1].highlightRequired &&
+                        'highlight-required'
+                    }`}
```

Comment:
This will add a spurious class name (`false`) when not required.

Reply:
Fixed in https://github.com/w3c/aria-at-app/pull/849/commits/ade11844c9e4b12cdd4f8eedacd46a5e302939a3